src/main/java/com/launchacademy/reviews/controllers/GenreRestController.java

```diff
@@ -38,4 +48,33 @@ public GenreRestController(GenreService genreService) {
     genreMap.put("genre", genreName);
     return genreMap;
   }
+
+  @PostMapping("/{id}/new")
+  public ResponseEntity saveNewFilm (@PathVariable Integer id, @Valid @RequestBody FilmForm filmForm, BindingResult bindingResult) {
+
+    if(bindingResult.hasErrors()) {
+      Map<String, Map<String, String>> errorsMap = new HashMap<>();
+      Map<String, String> errorsFieldMessage = new HashMap<>();
+      List<FieldError> errors = bindingResult.getFieldErrors();
+      for (FieldError error : errors ) {
+        errorsFieldMessage.put(error.getField(), error.getDefaultMessage());
+      }
+      errorsMap.put("errors", errorsFieldMessage);
+
+      return new ResponseEntity(errorsMap, HttpStatus.UNPROCESSABLE_ENTITY);
+    }
+    else {
+        Film film = new Film();
+        film.setTitle(filmForm.getTitle());
+        film.setYear(filmForm.getYear());
+        film.setImgUrl(filmForm.getImgUrl());
+        film.setDescription(filmForm.getDescription());
+        Genre genre = genreService.findById(filmForm.getGenreId()).get();
```

Comment:
Nice job creating a `FilmForm` to handle the mapping of the incoming data and then using that to create the Film object.  Since that filmForm will contain the genreId, it looks like the path variable `{id}` is never being used here.  One potential refactor (that you do not have to do at this time) would be to use and api endpoint of `api/v1/films` to handle the creation of a film.